packages/contracts/markets/base/reservesConfigs.ts

```diff
@@ -179,6 +195,21 @@ export const strategyVeloWETHDAI: IReserveParams = {
   reserveFactor: ethers.utils.parseUnits('0.15',18).toString(),
 };
 
+export const strategyVelowstETHWETH: IReserveParams = {
+  strategy: rateStrategyUnborrowable,
+  baseLTVAsCollateral: ethers.utils.parseUnits('0.65',18).toString(), 
+  liquidationThreshold: ethers.utils.parseUnits('0.7',18).toString(),
+  liquidationBonus: ethers.utils.parseUnits('1.1',18).toString(), 
+  borrowingEnabled: false,
+  reserveDecimals: '18', 
+  aTokenImpl: eContractid.AToken,
+  assetType: 5, //5 is enum for velo
+  supplyCap: "130",
+  borrowCap: '0',
+  borrowFactor: ethers.utils.parseUnits('1',18).toString(), //100% for now
```

Comment:
Does borrow factor matter when borrowing this asset is disabled?

also remind me, 100% means there is no extra collateral than normal needed when trying to borrow?

Reply:
no borrow factor does not matter. And yes you're right